### DISCUSSION TITLE
TotalDropout/more flexible Lambda

### DISCUSSION THREAD

**Discussion by AndrewAnnex** [albumentations-team/albumentations#1490] [Ideas]
I want to have more flexibility in what kinds of Lambda transforms are allowed in the library. In particular, I want to be able to do something like imgaug's TotalDropout where the image is returned as all zeros and all bboxes are deleted. I attempted to use the Lambda transform directly, but end up with an error with the apply_to_bbox/apply_to_bboxes functions. I think there should be a way to allow for dropping any and all labels.

**Comment by Dipet** [comment]
Try to inheriting from `DualTransform`. Look to `Crop` for example https://github.com/albumentations-team/albumentations/blob/master/albumentations/augmentations/crops/transforms.py#L108C12-L108C25
